integration_test/testutil_test.go

```diff
@@ -37,10 +38,28 @@ import (
 const (
 	persistentIndexCache = "krew-persistent-index-cache"
 	krewBinaryEnv        = "KREW_BINARY"
-	validPlugin          = "konfig" // a plugin in central index with small size
-	validPlugin2         = "mtail"  // a plugin in central index with small size
+	validPlugin          = "konfig"          // a plugin in central index with small size
```

Comment:
This list of (valid) external plugins is getting out of hand. Please don't add that many :pray:

Reply:
TBC new PR